scripts/configure-installer-tests-cluster.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Setup to find nessasary data from cluster setup
+## Constants
+HTPASSWD_FILE="./htpass"
+USERPASS="developer"
+HTPASSWD_SECRET="htpasswd-secret"
+# Overrideable information
+AUTH_DIR=${AUTH_DIR:-"$(pwd)/auth"}
+KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
+KUBEADMIN_PASSWORD_FILE_NAME=${KUBEADMIN_PASSWORD_FILE_NAME:-"kubeadmin-password"}
+KUBECONFIG_FILE_NAME=${KUBECONFIG_FILE_NAME:-"kubeconfig"}
```

Comment:
Why it is not using standard `KUBECONFIG`?

Reply:
> Why it is not using standard `KUBECONFIG`?

The actual variable exported is KUBECONFIG (), this is just an intermediary variable

Reply:
Hmm makes sense :)